test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1

```diff
@@ -2,20 +2,21 @@ Describe "SSH Remoting API Tests" -Tags "Feature" {
 
     Context "SSHConnectionInfo Class Tests" {
 
-        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" {
+        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" -Pending:$true {
 
+            # The test is pending because it is wrong or we have bug in code!
```

Comment:
I only add the comment for you. I don't mark it as pending. 
The code don't raise an exception as expected in the test (on windows). So we should open a new Issue.